java-client/flight-examples/build.gradle

```diff
@@ -1,12 +1,15 @@
 plugins {
     id 'application'
+    id 'java-library'
```

Comment:
If this is required for `api` then I would change it to `implementation` and leave this line out. Otherwise double check with Devin that this is an OK add.

Reply:
This is not typically appropriate, unless you want to have other consumers depend on flight-examples... I don't think that's the case here.

Reply:
per @devinrsmith, `implementation` is correct, will remove this line